circuit_maintenance_parser/parsers/aws.py

```diff
@@ -24,9 +24,29 @@ def parse_subject(self, subject):
         Example: AWS Direct Connect Planned Maintenance Notification [AWS Account: 00000001]
         """
         data = {}
-        search = re.search(r"\[AWS Account ?I?D?: ([0-9]+)\]", subject)
-        if search:
-            data["account"] = search.group(1)
+        # Common Subject strings for matching:
+        subject_map = [{"account": r"\[AWS Account ?I?D?: ([0-9]+)\]"}]
+
+        subject_list = []
```

Comment:
this is the exactly same code you have in the other function, extract this to a staticmethod to reuse